_data/schedules/25-summer.yml

```diff
@@ -10,4 +10,73 @@
     - time: "8:30"
       route_id: gas-works-locks-salmon-bay-loop
     - time: "10:00"
-      route_id: gas-works-lake-union-loop
\ No newline at end of file
+      route_id: gas-works-lake-union-loop
+
+- date: "2025-07-05"
+  plan:
+    - time: "8:30"
+      route_id: westlake-jack-perry-memorial-park-ob
+    - time: "9:30"
+      route_id: westlake-dravus-street-ob
+
+- date: "2025-07-12"
+  plan:
+    - time: "8:30"
+      route_id: locks-discovery-radar-magnolia-discovery-ob
+
+- date: "2025-07-19"
+  plan:
+    - time: "8:30"
+      route_id: northgate-cemeteries-interurban-boeing-creek-ob
+
+- date: "2025-07-26"
+  plan:
+    - time: "8:30"
+      route_id: cse-arb-lwb-seward-loop-cc
+
+- date: "2025-08-02"
+  plan:
+    - time: "8:30"
+      route_id: cap-hill-volunteer-park-westlake
+    - time: "9:10"
+      route_id: westlake-sculpture-marina-ob
+
+- date: "2025-08-09"
+  plan:
+    - time: "8:00"
+      route_id: beacon-i90-luther-burbank-ob
+
+- date: "2025-08-16"
+  plan:
+    - time: "8:30"
+      route_id: lynnwood-jogn-interurban-northgate
+
+- date: "2025-08-23"
```

Comment:
Another I90, also hard to get back from this one since it ends in Redmond. OTOH, epic point to point is good for summer?